client/elife-theme/src/index.js

```diff
@@ -3,51 +3,62 @@ import './fonts/index.css'
 
 export default {
   /* Colors */
-  colorBackground: '#fff',
   colorPrimary: '#0288d1',
-  colorSecondary: '#888',
+  colorSecondary: '#f7f7f7',
   colorFurniture: '#E0E0E0',
-  colorBorder: '#aaa',
+  colorBorder: '#E0E0E0',
   colorBackgroundHue:
-    '#f5f5f5' /* marginally darker shade of the app background so that it can be used to divide the interface when needed */,
+    '#f7f7f7' /* marginally darker shade of the app background so that it can be used to divide the interface when needed */,
   colorSuccess: '#629f43',
-  colorError: '#b50000',
+  colorError: '#cf0c4e',
+  colourWarning: '#e65100',
   colorText: '#212121',
-  colorTextReverse: '#fff',
+  colorTextSecondary: '#888',
   colorTextPlaceholder: '#bdbdbd',
+  colorTextReverse: '#fff',
 
   /* Text variables */
-  fontInterface: 'Georgia, serif',
-  fontHeading: '"Avenir Next Webfont", Arial, Helvetica, sans-serif',
-  fontReading: 'Georgia, serif',
-  fontWriting: '"Courier 10 Pitch", Courier, monospace',
+  fontInterface: '"Noto Sans", Arial, Helvetica, sans-serif',
+  fontHeading: '"Noto Sans SemiBold", Arial, Helvetica, sans-serif',
+  fontReading: '"Noto Serif", serif',
+  fontWriting: '"Noto Sans", Arial, Helvetica, sans-serif',
   fontSizeBase: '16px',
-  fontSizeBaseSmall: '13px',
-  fontSizeHeading1: '36px',
-  fontSizeHeading2: '26px',
-  fontSizeHeading3: '22px',
-  fontSizeHeading4: '20px',
-  fontSizeHeading5: '18px',
-  fontSizeHeading6: '16px',
-  fontLineHeight: '32px',
+  fontSizeBaseSmall: '14px',
+  fontSizeHeading1: '30px',
+  fontSizeHeading2: '22px',
+  fontSizeHeading3: '18px',
+  fontSizeHeading4: '16px',
+  fontSizeHeading5: '14px',
+  fontSizeHeading6: '12px',
+  fontLineHeightBase: '24px',
+  fontLineHeightBaseSmall: '18px',
+  fontLineHeight1: '36px',
+  fontLineHeight2: '30px',
+  fontLineHeight3: '24px',
+  fontLineHeight4: '24px',
+  fontLineHeight5: '18px',
+  fontLineHeight6: '16px',
 
   /* Spacing */
-  gridUnit: '24px',
-  subGridUnit: '6px',
+  gridUnit: '6px',
+  verticalXxl: 'gridUnit'*20,
+  verticalXl: 'gridUnit'*12,
+  verticalL: 'gridUnit'*8,
+  verticalM: 'gridUnit'*6,
+  verticalS: 'gridUnit'*4,
+  verticalXs: 'gridUnit'*2,
+  verticalXxs: 'gridUnit',
 
   /* Border */
   borderRadius: '3px',
   borderWidth: '1px',
   borderStyle: 'solid',
 
   /* Shadow (for tooltip) */
-  boxShadow: '0 2px 4px 0 rgba(51, 51, 51, 0.3)',
+  dropShadow: '0 2px 4px 0 rgba(0, 0, 0, 0.30)',
```

Comment:
FYI, I've raised [an issue](https://gitlab.coko.foundation/pubsweet/design/issues/29) in the pubsweet design repo about `boxShadow` vs `dropShadow`

Shouldn't have any impact on this PR though for now. Better that we use the same language that's in VARIABLES.md & @NickDuf's designs. But just worth noting that `boxShadow` is what's being used in the default pubsweet theme & presumably others.